planning/.pages

```diff
@@ -77,8 +77,8 @@ nav:
   - 'Additional Tools':
     - 'RTC Replayer': planning/rtc_replayer
     - 'Planning Debug Tools':
-      - 'About Planning Debug Tools': planning/planning_debug_tools
-      - 'Stop Reason Visualizer': planning/planning_debug_tools/doc-stop-reason-visualizer
+      - ['About Planning Debug Tools'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools)
+      - ['Stop Reason Visualizer'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools/doc-stop-reason-visualizer)
```

Comment:
This probably causes deploy-docs to fail.

Reply:
Thank you for the report.
Seems it was already fixed.
https://github.com/autowarefoundation/autoware.universe/pull/6465/files

Reply:
Yes, created an issue but forgot to link it here:
- https://github.com/autowarefoundation/autoware.universe/issues/6508